src/Sql/Sql/Database/Cmdlet/SetAzureSqlDatabase.cs

```diff
@@ -257,6 +268,7 @@ protected override IEnumerable<AzureSqlDatabaseModel> ApplyUserInputToModel(IEnu
                 LicenseType = LicenseType ?? model.FirstOrDefault().LicenseType, // set to original license type
                 AutoPauseDelayInMinutes = MyInvocation.BoundParameters.ContainsKey("AutoPauseDelayInMinutes") ? AutoPauseDelayInMinutes : (int?)null,
                 MinimumCapacity = MyInvocation.BoundParameters.ContainsKey("MinimumCapacity") ? MinimumCapacity : (double?)null,
+                ReadReplicaCount = MyInvocation.BoundParameters.ContainsKey("ReadReplicaCount") ? ReadReplicaCount : (int?)null,
```

Comment:
This works fine, but you can also use this pattern:https://github.com/Azure/azure-powershell/blob/e8ff3ad1e975135a616ec6a52bd4e0056bfc3892/src/Compute/Compute/Generated/Disk/Config/NewAzureRmDiskConfigCommand.cs#L157  (which uses an extension method and does not require strings for parameter names)

Reply:
Note that this is FYI, not a required change

Reply:
Done.  Please see latest.